detections/endpoint/windows_set_private_network_profile_via_registry.yml

```diff
@@ -0,0 +1,66 @@
+name: Windows Set Private Network Profile via Registry
+id: a277acde-9bfd-4edb-b201-7cfc504003e2
+version: 1
+date: '2025-08-06'
+author: Teoderick Contreras, Splunk
+status: production
+type: Anomaly
+description: The following analytic detects attempts to modify the Windows Registry to change a network profile's category to "Private", which may indicate an adversary is preparing the environment for lateral movement or reducing firewall restrictions. Specifically, this activity involves changes to the Category value within the HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion\NetworkList\Profiles\{GUID} registry path. A value of 1 corresponds to a private network profile, which typically enables less restrictive firewall policies. While this action can occur during legitimate network configuration, it may also be a sign of malicious behavior when combined with other indicators such as suspicious account activity, unexpected administrative privilege usage, or execution of unsigned binaries. Monitoring for this registry modification—especially outside standard IT processes or correlated with persistence mechanisms—can help identify stealthy post-exploitation activity.
+data_source:
+  - Sysmon EventID 13
+search: '| tstats `security_content_summariesonly` count min(_time) as firstTime max(_time) as lastTime FROM datamodel=Endpoint.Registry 
+  WHERE (Registry.registry_value_name= "Category" Registry.registry_path = "*\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\NetworkList\\Profiles*") Registry.registry_value_data = 0x00000000
```

Comment:
Do we know if the adversary used reg or PowerShell in this case? Because perhaps the default behavior of this is that svchost make that change but an attacker might use a tool.

Of course a GUI could still be used but just thinking out loud to try and make this more accurate.

Perhaps a VT search for this registry change would bring good results.